tests/cli/commands/test_config_command.py

```diff
@@ -43,3 +43,10 @@ def test_cli_show_config_should_display_key(self):
             config_command.show_config(self.parser.parse_args(['config']))
         self.assertIn('[core]', temp_stdout.getvalue())
         self.assertIn('testkey = test_value', temp_stdout.getvalue())
+
+    @mock.patch.dict("os.environ", {"AIRFLOW__CORE__DAGS_FOLDER": "/tmp/test_folder"})
+    def test_cli_show_config_should_respect_env_variable(self):
+        with contextlib.redirect_stdout(io.StringIO()) as temp_stdout:
+            config_command.show_config(self.parser.parse_args(['config']))
+        self.assertIn('[core]', temp_stdout.getvalue())
+        self.assertIn('dags_folder = /tmp/test_folder', temp_stdout.getvalue())
```

Comment:
This test duplicates/tests the exact same thing as your test in tests/test_configuration.py -- we don't need to cover the same code twice.

Reply:
Removed.